kieserver/image.yaml

```diff
@@ -189,6 +189,71 @@ envs:
     - name: "DROOLS_SERVER_FILTER_CLASSES"
       example: "true"
       description: "KIE execution server class filtering (Sets the org.drools.server.filter.classes system property)"
+    - name: "KIE_SERVER_MGMT_DISABLED"
```

Comment:
AFAIK this property shouldn't be here as it was deleted. For more info please see RHDM-587.